tests/third_party/cupy/sorting_tests/test_search.py

```diff
@@ -366,7 +366,6 @@ def test_argwhere(self, xp, dtype):
 # return cupy.nonzero(self.array)
 
 
-@testing.gpu
 class TestNanArgMin(unittest.TestCase):
     @testing.for_all_dtypes(no_complex=True)
```

Comment:
also cupy doesn't have `accept_error=ValueError` in `numpy_cupy_allclose`, do we need that?